src/main/java/ch/njol/skript/expressions/ExprLocation.java

```diff
@@ -35,23 +35,23 @@
 import ch.njol.skript.util.Direction;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Location")
 @Description("The location where an event happened (e.g. at an entity or block), or a location <a href='#ExprDirection'>relative</a> to another (e.g. 1 meter above another location).")
-@Examples({"drop 5 apples at the event-location # exactly the same as writing 'drop 5 apples'",
-		"set {_loc} to the location 1 meter above the player"})
+@Examples({
+	"drop 5 apples at the event-location # exactly the same as writing 'drop 5 apples'",
+	"set {_loc} to the location 1 meter above the player"
+})
 @Since("2.0")
 public class ExprLocation extends WrapperExpression<Location> {
+
 	static {
 		Skript.registerExpression(ExprLocation.class, Location.class, ExpressionType.SIMPLE, "[the] [event-](location|position)");
-		Skript.registerExpression(ExprLocation.class, Location.class, ExpressionType.COMBINED, "[the] (location|position) %directions% [%location%]");
+		Skript.registerExpression(ExprLocation.class, Location.class, ExpressionType.COMBINED, "[the] (location|position) %directions% [%locations%]");
```

Comment:
```suggestion
		Skript.registerExpression(ExprLocation.class, Location.class, ExpressionType.COMBINED, "[the] (location|position)[s] %directions% [%locations%]");
```

Also add `s` depending on plurality to the toString